ilastik/applets/dataSelection/opDataSelection.py

```diff
@@ -140,12 +140,16 @@ def from_h5_group(cls, data: h5py.Group, params: Dict = None):
         params = params or {}
         params.update(
             {
-                "axistags": AxisTags.fromJSON(data["axistags"][()].decode("utf-8")),
                 "allowLabels": data["allowLabels"][()],
                 "nickname": data["nickname"][()].decode("utf-8"),
                 "project_file": data.file,
             }
         )
+        if "axistags" in data:
+            params["axistags"] = AxisTags.fromJSON(data["axistags"][()].decode("utf-8"))
+        elif "axisorder" in data:  # legacy support
+            axisorder = data["axisorder"][()].decode("utf-8")
+            params["axistags"] = vigra.defaultAxistags(axisorder)
```

Comment:
Small nitpick, I would add an empty line after `elif` block to be clearer that it's separate if statement.

Reply:
sure thing